### DISCUSSION TITLE
client有distcc字样，但是service端没有看到网络\CPU有变化，似乎没有执行远程\分布式编译

### DISCUSSION THREAD

**Discussion by yujunting** [xmake-io/xmake#5638] [Q&A]
![image](https://github.com/user-attachments/assets/2b97a0e4-6d31-4714-98b5-e72799befb09)

**Comment by waruqi** [comment]
<img width="216" alt="image" src="https://github.com/user-attachments/assets/8072eb8f-165e-4994-b2bf-3314ffdd9802">

这里的就是走的 distcc 的，但是二次变异有本地 cache，也会尝试优先走本地cache， 不再走远程编译

**Reply by yujunting** [reply]
整个编译过程没有看到远端service有CPU和内存的变化，怎么确认参与了编译？

**Reply by yujunting** [reply]
<remote_build_server>: listening 0.0.0.0:9691 ..
<remote_cache_server>: listening 0.0.0.0:9692 ..
<distcc_build_server>: listening 0.0.0.0:9693 ..
<distcc_build_server>: <socket: tcp4/968.0>: session connected
<distcc_build_server>: <socket: tcp4/968.0>: session end
这个是service的日志
